samples/detect.v1p4beta1.js

```diff
@@ -0,0 +1,219 @@
+    // Supported mime_types are: 'application/pdf' and 'image/tiff'
+    mimeType: 'application/pdf',
+    gcsSource: {
+      uri: gcsSourceUri,
+    },
+  };
+  const features = [{type: 'DOCUMENT_TEXT_DETECTION'}];
+  const request = {
+    requests: [
+      {
+        inputConfig: inputConfig,
+        features: features,
+        // Annotate the first two pages and the last one (max 5 pages)
+        // First page starts at 1, and not 0. Last page is -1.
+        pages: [1, 2, -1]
+      },
+    ],
+  };
+
+  const [result] = await client.batchAnnotateFiles(request);
+  const responses = result.responses[0].responses;
+
+  responses.forEach(response => {
+    response.fullTextAnnotation.pages.forEach(page => {
+      page.blocks.forEach(block => {
+        console.log(`\nBlock confidence: ${block.confidence}`);
+        block.paragraphs.forEach(paragraph => {
+          console.log(`\tParagraph confidence: ${paragraph.confidence}`);
+          paragraph.words.forEach(word => {
+            var symbol_texts = [];
+            for (var i = 0; i < word.symbols.length; i++) {
+              symbol_texts.push(word.symbols[i].text)
+            }
+            const word_text = symbol_texts.join('');
+            console.log(`\t\tWord text: ${word_text} (confidence: ${word.confidence})`);
+            word.symbols.forEach(symbol => {
+              console.log(`\t\t\tSymbol: ${symbol.text} (confidence: ${symbol.confidence})`);
+            })
+          })
+        })
+      })
+    })
+  });
+  // [END vision_fulltext_detection_pdf_gcs_beta]
+}
+
+async function detectBatchAnnotateImageUri(inputImageUri, outputUri) {
+  // [START vision_async_batch_annotate_images_beta]
+
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p4beta1;
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // GCS path where the image resides
+  // const inputImageUri = 'gs://my-bucket/my_image.jpg';
+  // GCS path where to store the output json
+  // const outputUri = 'gs://mybucket/out/'
+
+  const features = [
+    {type: 'DOCUMENT_LABEL_DETECTION'},
+    {type: 'DOCUMENT_TEXT_DETECTION'},
+    {type: 'DOCUMENT_IMAGE_DETECTION'},
+  ];
+
+  const outputConfig = {
+    gcsDestination: {
+      uri: outputUri,
+    },
+  };
+
+  const request = {
+    requests: [
+      {
+        image: {
+          source: {
+            imageUri: inputImageUri,
+          },
+        },          
+        features: features,
+      },
+    ],
+    outputConfig: outputConfig,
+  };
+
+  const [operation] = await client.asyncBatchAnnotateImages(request);
+  const [filesResponse] = await operation.promise();
+
+  const destinationUri =
+    filesResponse.outputConfig.gcsDestination.uri;
+  console.log('Json saved to: ' + destinationUri);
+  // [END vision_async_batch_annotate_images_beta]
+}
+
+require(`yargs`)
```

Comment:
As per the new rubric, I don't think we're using `yargs` anymore for these

Reply:
So, what should I do? Can you provide me a link to a sample with the new rubric in mind?

Reply:
Sure!  You can find an example here:
https://github.com/googleapis/nodejs-dlp/pull/154